src/org/centos/pipeline/PipelineUtils.groovy

```diff
@@ -955,7 +955,7 @@ def setupContainerTemplates(String openshiftProject) {
     openshift.withCluster() {
         openshift.withProject(openshiftProject) {
             dir('config/s2i') {
-                sh './create-containers.sh'
+                sh 'bash -x ./create-containers.sh'
```

Comment:
We can merge this, but I don't really see the point. Functionally, this is not going to change anything, so there is no way that this is currently blocking LP release from working and that merging it would fix it. It will just give us more output in the jenkins logs. It does not matter much to me if we merge it